packages/blocks/src/api/factory.js

```diff
@@ -279,21 +280,22 @@ export function findTransform( transforms, predicate ) {
  * transform object includes `blockName` as a property.
  *
  * @param {string}  direction Transform direction ("to", "from").
- * @param {?string} blockName Optional block name.
+ * @param {string|Object} blockTypeOrName  Block type or name.
  *
  * @return {Array} Block transforms for direction.
  */
-export function getBlockTransforms( direction, blockName ) {
+export function getBlockTransforms( direction, blockTypeOrName ) {
 	// When retrieving transforms for all block types, recurse into self.
-	if ( blockName === undefined ) {
+	if ( blockTypeOrName === undefined ) {
 		return flatMap(
 			getBlockTypes(),
 			( { name } ) => getBlockTransforms( direction, name )
```

Comment:
This could be `partial( getBlockTransforms, direction )`. I dunno that I'd care to change it though (given `partial`'s questionable future)